src/wx/strutils.h

```diff
@@ -7,15 +7,22 @@
 #include <wx/string.h>
 #include <wx/arrstr.h>
 
+namespace strutils {
+
 // From: https://stackoverflow.com/a/7408245/262458
-wxArrayString str_split(const wxString& text, const wxString& sep, bool empty_token_is_sep=false);
+wxArrayString split(const wxString& text, const wxString& sep, bool empty_token_is_sep=false);
 
 // Same as above, but it includes the sep dir.
 // If "A,,,B" is the text and "," is sep, then
 // 'A', ',' and 'B' will be in the output.
-wxArrayString str_split_with_sep(const wxString& text, const wxString& sep);
+wxArrayString split_with_sep(const wxString& text, const wxString& sep);
+
+wxString trim(const wxString& in_str);
+
+wxString remove_word(const wxString& in_str, const wxString& word);
+
+wxString titlecase(const wxString& in_str);
```

Comment:
These are no longer used. We could still keep them around in case we need them down the line but I think we're better off removing dead code.